tests/scripts/analyze_outcomes.py

```diff
@@ -388,6 +388,12 @@ class DriverVSReference_ecp_light_only(outcome_analysis.DriverVSReference):
             re.compile(r'ECP point multiplication .*'),
             re.compile(r'ECP test vectors .*'),
         ],
+        # !!!WARNING!!!!
+        # This is TEMPORARY and MUST be removed once psa#299 and mbedtls#10187
+        # will be merged.
```

Comment:
If I've understood correctly, this will not be able to be temporary. This part of `analyze_outcomes.py` checks for discrepancies between which testcases are executed with drivers vs the builtin implementation.

Going by the commit message of [876bad84](https://github.com/Mbed-TLS/TF-PSA-Crypto/pull/299/commits/867bad84ba7405ea2b1e3ecb30272e20c28a7c70), restartable key generation is only supported via the builtin driver even after we merge Mbed-TLS/TF-PSA-Crypto#299. If this is the case, `analyze_outcomes.py` will always need this exception.

What has changed to make it unsupported through a driver?

Reply:
Oh wait, you are right. When I wrote that comment I was thinking about the future case when we'll be testing and merging this PR while psa#299 is still not merged. In that case there will be no disparity issue, but once psa#299 will be in, disparities will be there.
Sorry I got confused about that when writing the comments. I will update them accordingly.